TestingGround/FramerateStabilizer/obj/Debug/netcoreapp2.2/FramerateStabilizer.csproj.FileListAbsolute.txt

```diff
@@ -14,3 +14,11 @@ H:\Programming Stuff\Side-Projects\TestingGround\FramerateStabilizer\obj\Debug\n
 H:\Programming Stuff\Side-Projects\TestingGround\FramerateStabilizer\obj\Debug\netcoreapp2.2\FramerateStabilizer.AssemblyInfo.cs
 H:\Programming Stuff\Side-Projects\TestingGround\FramerateStabilizer\obj\Debug\netcoreapp2.2\FramerateStabilizer.dll
 H:\Programming Stuff\Side-Projects\TestingGround\FramerateStabilizer\obj\Debug\netcoreapp2.2\FramerateStabilizer.pdb
+H:\ProgrammingStuff\Side-Projects\TestingGround\FramerateStabilizer\bin\Debug\netcoreapp2.2\FramerateStabilizer.deps.json
```

Comment:
These are all generated files. We should remove them from the repo

Reply:
I'm not sure how to remove these

Reply:
I can see about that. It’s a matter of git rm but have to be careful